frontend/src/app/config/gloves-config.ts

```diff
@@ -5,10 +5,10 @@ import {Config} from './config';
  */
 export const CONFIG: Config = new Config(
     'Blue Gloves 4 All',
-    'www.gloveygloversonsgloveemporium.org',
-    80,
+    'https://bluegloves.h2ms.org',
+    443,
     '/assets/images/banners/gloves-banner.png',
     '/assets/images/logos/h2ms-logo.png',
-    'https://test.h2ms.org',
+    'https://bluegloves.h2ms.org',
     443,
     [{id: null, displayName: null}]); // todo: add gloves favicon and logo
```

Comment:
This line actually has to do with the reports.

Could we change it to 
`[{id: 3, displayName: "Gloves removed?"}] `

So the reports page can find the question